attributes/default.rb

```diff
@@ -44,7 +44,7 @@
 default['chef_client']['log_dir']     = '/var/log/chef'
 
 # If log file is used, default permissions so everyone can read
-default['chef_client']['log_perm'] = '640'
+default['chef_client']['log_perm'] = '0644'
```

Comment:
We probably don't want 644 for the chef-client log. That's a pretty significant security concern given what's in that log file.

Reply:
You are correct. That's probably copy-paste error from `template` resource :)

Reply:
With `640` `Users` can read `client.log` file.

![2018-09-17 7 51 33](https://user-images.githubusercontent.com/2243323/45606541-a0a23300-ba4e-11e8-83d6-28e3a12d110a.png)
 
Without `owner` and `group` `Users` can read `client.log` also.

![2018-09-17 7 55 48](https://user-images.githubusercontent.com/2243323/45606613-42c21b00-ba4f-11e8-9309-4f149f4f540d.png)

@tas50 What do you think?

Reply:
On linux platform
![2018-09-17 8 12 39](https://user-images.githubusercontent.com/2243323/45606918-b49b6400-ba51-11e8-9fcf-a535d524cdc9.png)

BTW I wil fix comment in this PR as everyone can't read `client.log` on Linux. Only `root` and `root` group.
https://github.com/chef-cookbooks/chef-client/blob/master/attributes/default.rb#L46